src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeControllerTest.java

```diff
@@ -184,11 +183,10 @@ public void should_successfully_return_all_envelopes_with_processed_status_for_a
             .andExpect(MockMvcResultMatchers.jsonPath("envelopes[0].id").exists());
 
         assertThat(envelopeRepository.findAll())
-            .hasSize(2)
+            .hasSize(1)
             .extracting("zipFileName", "status")
             .containsExactlyInAnyOrder(
-                tuple("7_24-06-2018-00-00-00.zip", PROCESSED),
-                tuple("8_24-06-2018-00-00-00.zip", METADATA_FAILURE)
```

Comment:
Looks like `METADATA_FAILURE` status can be remove as envelopes in such state will never be created after this change. (and such envelopes can be deleted from db).

Ok for next PR imho.